pc/manager.cpp

```diff
@@ -743,6 +750,13 @@ void manager::on_response( rpc::get_slot *res )
     clnt_.send( breq_ );
   }
 
+  if (has_status( PC_PYTH_RPC_CONNECTED ) && !is_pub_) {
+    // Flush any partial batches of updates, as we've reached the end of the list of products.
+    for( user *uptr = olist_.first(); uptr; uptr = uptr->get_next() ) {
+      uptr->send_pending_upds(uptr->num_pending_upds());
+    }
```

Comment:
It might be nice to factor this loop out into an `manager::send_pending_upds` function.